Packs/CommonReports/Reports/report-last7DaysIncidentReport.json

```diff
@@ -838,7 +838,7 @@
   },
   "endingDate": "0001-01-01T00:00:00Z",
   "sensitive": false,
-  "recipients": null,
+  "recipients": [],
```

Comment:
what's the impact of this

Reply:
that is a good question, the schema in the sdk required an array here and I could not find a way to make it accept null. will try and figure if this is a problem.

Reply:
let's just upload them and see if we get different results

Reply:
checked - worked the same as the existing one